packages/driver/cypress/integration/commands/clock_spec.js

```diff
@@ -125,6 +125,14 @@ describe('src/cy/commands/clock', () => {
       })
     })
 
+    it('overwrite gracefully', () => {
```

Comment:
Let's be more explicit about what we're testing here. Something like:

```suggestion
    it('overwrites without crashing', () => {
```